lib/model/DebugLogModel.class.php

```diff
@@ -46,11 +46,11 @@ public function log($class, $function, array $args = array(), $dump = null) {
 		if (LoginModel::instance()->isSued()) {
 			$entry->su_uid = LoginModel::getSuedFrom()->uid;
 		}
-		$entry->ip = $_SERVER['REMOTE_ADDR'];
-		$entry->referer = $_SERVER['HTTP_REFERER'];
-		$entry->request = REQUEST_URI;
-		$entry->ip_referer = HTTP_REFERER;
-		$entry->user_agent = $_SERVER['HTTP_USER_AGENT'];
+		$entry->ip = @$_SERVER['REMOTE_ADDR'] ?: '0.0.0.0';
+		$entry->referer = @$_SERVER['HTTP_REFERER'] ?: 'CLI';
+		$entry->request = REQUEST_URI ?: 'CLI';
+		$entry->ip_referer = HTTP_REFERER ?: '';
+		$entry->user_agent = @$_SERVER['HTTP_USER_AGENT'] ?: 'CLI';
```

Comment:
Dit gedeelte is om warnings bij het runnen van commandline scripts te voorkomen.

Reply:
In dat geval is 127.0.0.1 duidelijker als ip